### PR TITLE
fix: import log to destination project

### PR DESCRIPTION
**Pull request by minherz**
## Description

Patch logName of imported logs to use destination project.
Store original logName in user labels.
Refactor method comments.
Refactor handling partial errors of type WriteLogEntriesPartialErrors.

### CODE REVIEW

File: logging/import-logs/main.py [hunk 1]
```diff
@@ -70,24 +56,28 @@ def eprint(*objects: str, **kwargs: TypedDict) -> None:
 
 
 def _day(blob_name: str) -> int:
-    """Parse day number from Blob's name
-    using the following Blob name convention:
-    <LOG_ID>/YYYY/MM/DD/<OBJECT_NAME>
+    """Parse day number from Blob's path
+
+    Use the known Blob path convention to parse the day part from the path.
+    The path convetion is <LOG_ID>/YYYY/MM/DD/<OBJECT_NAME>
```
Comment (kweinmeister) [inline logging/import-logs/main.py:62]: typo: convention

Comment (minherz) [reply logging/import-logs/main.py:62]: apologies, i think tha the word "convention" is spelled correctly. do you propose replacing the word "convention" with another word? if you do, can you propose a replacement?

Comment (m-strzelczyk) [inline logging/import-logs/main.py:62]:
```suggestion
    The path convention is <LOG_ID>/YYYY/MM/DD/<OBJECT_NAME>
```

**Review by m-strzelczyk — APPROVED** [review]

File: logging/import-logs/main.py (hunk 1 above)
Comment (m-strzelczyk) [reply logging/import-logs/main.py]: There was 'n' missing at line 62 :) I added it.

**Pull request merged**